howso/client/exceptions.py

```diff
@@ -1,4 +1,23 @@
-import json
+from collections.abc import Generator, Mapping
```

Comment:
Probably not needed, but in case it was merely overlooked, consider adding `from __future__ import annotations`.